File: api_refurbish/src/models/Localization.js
const { Model, DataTypes } = require('sequelize');

class Localization extends Model {

    static init(sequelize) {
        super.init({
            address: DataTypes.STRING,
            created_by: DataTypes.STRING,
            updated_by: DataTypes.STRING,
        },
            { sequelize }
        )
    }

    static associate(models) {
        this.belongsToMany(models.Part, { foreignKey: 'localization_id', through: 'part_localizations', as: 'parts' });
    }
}

module.exports = Localization;

File: web_refurbish/lib/model/Localization.dart
class Localization {
  int _id;
  String _address;
  String _created_by;

  int get id => _id;

  set id(int value) {
    _id = value;
  }

  Localization(this._address);

  Localization.fromJson(Map<String, dynamic> json)
      : this._id = json['id'],
        this._address = json['address'];

  Map<String, dynamic> toJson() =>
      {"address": this._address, "created_by": this._created_by};

  String get address => _address;

  set address(String value) {
    _address = value;
  }

  String get created_by => _created_by;

  set created_by(String value) {
    _created_by = value;
  }
}
